csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs

```diff
@@ -758,7 +758,7 @@ private void TestModelSequenceOfMapStringFloat()
             }
         }
 
-        [Fact(Skip = "The Model Serialization Test fails on linux. Test skipped until fixed. Serialization API should not be used before fix.")]
```

Comment:
@shahasad : You had disabled this test in PR #1688 What was the exact issue? Was the test flaky or consistently failing?
This PR just enables the test again since now the failures are not observed in the CI (relying on the fact that some change between the disable time and now fixed the issue)

Reply:
failure is only observed on the NuGet pipeline, because dotnet Linux test is run only there. So, make sure the NuGet package CPU build passes with the branch, then it's good to merge.

Reply:
The Linux CI build did run the test and it succeeded:

[ RUN      ] InferenceSessionTests.TestModelSerialization
2: 2019-10-21 20:17:54.056861594 [W:onnxruntime:InferenceSessionTests.TestModelSerialization, inference_session.cc:583 Initialize] Serializing Optimized ONNX model with Graph Optimization level greater than 2 is not supported.
2: [       OK ] InferenceSessionTests.TestModelSerialization (38 ms)

Reply:
Spoke to Asad offline - this is not the C# test. Scheduled nuget packaging pipeline for this PR as the C# test only runs as part of that build